test/tools/isoltest.cpp

```diff
@@ -401,20 +401,26 @@ int main(int argc, char const *argv[])
 {
 	setupTerminal();
 
-	solidity::test::IsolTestOptions options(&TestTool::editor);
-
-	try
 	{
-		if (options.parse(argc, argv))
-			options.validate();
-		else
+		using IOptions = solidity::test::IsolTestOptions;
+		std::unique_ptr<IOptions> options(new IOptions(&TestTool::editor));
```

Comment:
```suggestion
		auto options = std::make_unique<solidity::test::IsolTestOptions>(&TestTool::editor);
```

Away with the ``new``s ;-D!!!